src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -162,6 +162,9 @@
 			<x:Double x:Key="MaterialOutlinedTextBoxMinHeight">56</x:Double>
 			<Thickness x:Key="MaterialOutlinedTextBoxBorderPadding">1</Thickness>
 			<x:Double x:Key="MaterialOutlinedTextBoxBorderThicknessFocused">2</x:Double>
+			<Thickness x:Key="MaterialOutlinedTextBoxPaddingFocused">0</Thickness>
+			<x:Double x:Key="MaterialOutlinedTextBoxBorderThicknessPointerOver">2</x:Double>
+			<Thickness x:Key="MaterialOutlinedTextBoxPaddingPointerOver">0</Thickness>
 			<!--#endregion-->
 		</ResourceDictionary>
 
```

Comment:
@Kunal22shah the related documentation will also need to be verify and adjusted as needed please
https://github.com/unoplatform/uno.themes/blob/master/doc/styles/TextBox.md